eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/GovSpec.hs

```diff
@@ -785,12 +828,123 @@ secondHardForkCantFollow = do
         , pProcGovAction = HardForkInitiation SNothing protver1
         , pProcAnchor = def
         }
+  let
+    govAction = HardForkInitiation (SJust (GovPurposeId gaid1)) protver2
   submitFailingProposal
     ( ProposalProcedure
         { pProcDeposit = pp ^. ppGovActionDepositL
         , pProcReturnAddr = rewardAccount
-        , pProcGovAction = HardForkInitiation (SJust (GovPurposeId gaid1)) protver2
+        , pProcGovAction = govAction
         , pProcAnchor = def
         }
     )
-    [inject (ProposalCantFollow @era (SJust (GovPurposeId gaid1)) protver2 protver1)]
+    [ inject $ MalformedProposal govAction
+    ]
+
+committeeMemberVotingOnCommitteeChange ::
+  forall era.
+  ( ConwayEraImp era
+  , GovState era ~ ConwayGovState era
+  , Inject (ConwayGovPredFailure era) (PredicateFailure (EraRule "LEDGER" era))
+  ) =>
+  ImpTestM era ()
+committeeMemberVotingOnCommitteeChange = do
+  (_, ccHot) <- electBasicCommittee
```

Comment:
I think it would make sense for `electBasicCommittee` to also return `prevGovActionId`, so we don't have to look it up in the state. I can do this change at some later point, if you'd rather not deal with it.